README.md

```diff
@@ -36,21 +36,47 @@ $ make
 ```
 This builds the plugin in `/build/rootfs`
 
+Unit testing:
+```
+$ make test-small
+```
+
 ### Configuration and Usage
 * Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 * Ensure `$SNAP_PATH` is exported
 `export SNAP_PATH=$GOPATH/src/github.com/intelsdi-x/snap/build`
 
 ## Documentation
-This Snap processor plugin calculates statistics over a sliding window. Currently, the plugin calculates the mean, median, standard deviation, variance, 95th-percentile and 99th-percenticle over a sliding window. 
-
-Note: This Snap processor plugin changes the metric data type to `map[string]float64`. Any Snap publisher plugin used with this plugin should register the data type with gob. 
-
-```
-import "encoding/gob"
-
-gob.Register(map[string]float64{})
-```
+This Snap processor plugin calculates statistics over a sliding window. Currently, the plugin calculates the following statistics over a sliding window:
+1) Count - length of the buffer
+2) Sum - total sum of data values in the buffer
+3) Mean - average of data values 
+4) Median - middle data value in a sorted buffer 
+5) Minimum - minimum of all data values  
+6) Maximum - maximum of all data values 
+7) Range - difference of largerst and smallest values 
+8) Standard deviation - measure that quantifies the amount of varitation of a dataset 
+9) Variance - measures how far a set of numbers are spread out from their mean
+10) Mode - the number which appears most often in a dataset
+11) Kurtosis - measures the degree of peakedness of a distribution
```

Comment:
give more specific explanation so that users know how this is calculated without reading code.